src/Common/src/System/Net/HttpKnownHeaderNames.TryGetHeaderName.cs

```diff
@@ -8,6 +8,9 @@ namespace System.Net
 {
     internal static partial class HttpKnownHeaderNames
     {
+        private const string Gzip = "gzip";
```

Comment:
Yes, there is a ton more cleanup that can and should be done.  Note that in this case, I'm not actually increasing the number of duplicates... this duplicate already existed, I've just moved the file it was in.